src/solver/hydro/management/management.cpp

```diff
@@ -400,25 +394,24 @@ void HydroManagement::prepareNetDemand(uint numSpace)
             // Aggregated renewable production: wind & solar
             if (parameters_.renewableGeneration.isAggregated())
             {
-                netdemand = +scratchpad.ts.load[ptchro.Consommation][hour]
-                            - scratchpad.ts.wind[ptchro.Eolien][hour] - scratchpad.miscGenSum[hour]
-                            - scratchpad.ts.solar[ptchro.Solar][hour] - ror[hour]
+                netdemand = +area.load.series->getCoefficient(year, hour)
+                            - area.wind.series->getCoefficient(year, hour) - scratchpad.miscGenSum[hour]
+                            - area.solar.series->getCoefficient(year, hour)
```

Comment:
Here you could use the version that returns a `double*`, so you'd only need 1 function call instead of 8760